src/Build/BackEnd/Components/SdkResolution/SdkResolverService.cs

```diff
@@ -465,10 +487,21 @@ private void RegisterResolversManifests(ElementLocation location)
                 {
                     return;
                 }
+                IReadOnlyList<SdkResolverManifest> allResolversManifests;
+#if DEBUG
+                if (_fake_initialization)
+                {
+                    allResolversManifests = _fakeManifestRegistry;
+                }
+                else
+                {
+#endif
+                    allResolversManifests = _sdkResolverLoader.GetResolversManifests(location);
+#if DEBUG
+                }
+#endif
```

Comment:
I've reduced it to enclose an if-branch that I need removed for non-debug environment.
Otherwise build complains about stuff not being in scope (since the static definitions for the test are not there and I need at least the _fake_initialization one to insert the thread.sleeps)